cluster/cluster.yaml

```diff
@@ -37,8 +37,6 @@ Resources:
       Tags:
         - Key: 'kubernetes.io/cluster/{{.Cluster.ID}}'
           Value: owned
-        - Key: 'kubernetes:application'
-          Value: kube-ingress-aws-controller
```

Comment:
We may depend on this in the ingress-controller

Reply:
I see, I'll restore this.